juriscraper/opinions/united_states/territories/visuper_p.py

```diff
@@ -0,0 +1,37 @@
+# Scraper for Superior Court of the Virgin Islands (published)
+# CourtID: visuper_p
+
+from juriscraper.OpinionSiteLinear import OpinionSiteLinear
+
+
+class Site(OpinionSiteLinear):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.court_id = self.__module__
+        self.url = (
+            "https://superior.vicourts.org/court_opinions/published_opinions"
+        )
+        self.request["verify"] = False
```

Comment:
why did you add verify false?

Reply:
I've used the example you mention, that's why I thought for Linear Sites were needed but looking now more deeply I think is no needed to deactivate de SSL verification in the request because there is no error when executing in local. Fixed

Reply:
That makes sense. Thanks for explaining it.